fdbclient/NativeAPI.actor.cpp

```diff
@@ -1611,6 +1611,28 @@ DatabaseContext::DatabaseContext(Reference<AsyncVar<Reference<IClusterConnection
 		    std::make_unique<FaultToleranceMetricsImpl>(
 		        singleKeyRange("fault_tolerance_metrics_json"_sr)
 		            .withPrefix(SpecialKeySpace::getModuleRange(SpecialKeySpace::MODULE::METRICS).begin)));
+		registerSpecialKeysImpl(
+		    SpecialKeySpace::MODULE::BULKLOADING,
+		    SpecialKeySpace::IMPLTYPE::READONLY,
+		    std::make_unique<BulkLoadStatusImpl>(
+		        KeyRangeRef("status/"_sr, "status0"_sr)
+		            .withPrefix(SpecialKeySpace::getModuleRange(SpecialKeySpace::MODULE::BULKLOADING).begin)));
+		registerSpecialKeysImpl(
+		    SpecialKeySpace::MODULE::BULKLOADING,
+		    SpecialKeySpace::IMPLTYPE::READWRITE,
+		    std::make_unique<BulkLoadTaskImpl>(
+		        KeyRangeRef("task/"_sr, "task0"_sr)
+		            .withPrefix(SpecialKeySpace::getModuleRange(SpecialKeySpace::MODULE::BULKLOADING).begin)));
+		registerSpecialKeysImpl(
+		    SpecialKeySpace::MODULE::BULKLOADING,
+		    SpecialKeySpace::IMPLTYPE::READWRITE,
+		    std::make_unique<BulkLoadCancelImpl>(
+		        KeyRangeRef("cancel/"_sr, "cancel0"_sr)
+		            .withPrefix(SpecialKeySpace::getModuleRange(SpecialKeySpace::MODULE::BULKLOADING).begin)));
+		registerSpecialKeysImpl(SpecialKeySpace::MODULE::BULKLOADING,
```

Comment:
Formatting is not same as that of the previous three registrations.

READONLY is in separate arena but cancelled items are not? Is there a reason for this layout? How the special keys are accessed?